error handling
error: anError
	^ self new
		error: anError